Improvement to the dbrowse Script

amarakon opened the pull request:
I added useful options to the dbrowse script to make it a suckless file manager.
The main feature of it is that you can pass the `-p` or `--program` option to open the default program for the selection.
For example, if you select a `.png` file, your default image viewer will be used.
The `-c` or `--copy` option allows you to copy the contents of a file.
The `-cf` or `--copy-file` option allows you to copy a specific line.
Here is the help message invoked by running `dbrowse -h` or `dbrowse --help`:

```
Usage:	dfm [options] [target] [prompt]

Options:
 -r|--raw       │ Get the raw output of the selection
 -c|--copy      │ Copy the contents of the selection
-cf|--copy-file │ Copy a line from the contents of the selection
 -p|--program   │ Open the appropriate program for the selection
 -h|--help	│ Print this help message and exit

When no arguments are supplied, the target and prompt will be the working directory, and the program option will be used.

```

clamiax commented:
I noticed, among the other things, that you hid `..` from the list, you hardcoded your home directory and used `dfm` in the usage section. I think this PR is not suitable for a simple script as `dbrowse`, you better fork or mantain it as a separate project (dfm sounds nice).

amarakon commented:
I fixed the hard-coded home directory to make it work on any home directory. I also changed the name from dfm to dbrowse. It should be good for anyone to use it now. But my question is: is it better to fork your entire `scripts` repository or to create a new repository called `dfm` and give credit to you for the script?

clamiax commented:
I suggest making an ad-hoc repository only for the `dfm` project. I'm glad you got inspired by `dbrowse` but please don't feel obligated in any way. However if you wish to credit me then I will appreciate. Thanks!

amarakon commented:
Okay that's what I will do, thank you.

vitaly-zdanevich commented:
If anybody is looking for that fork: https://github.com/amarakon/dfm
